test/index.html

```diff
@@ -86,7 +91,23 @@
   refresh();
 });
 
+function getQueryVariable(variable)
```

Comment:
Is this really the cleanest way to do this? Is there no direct API to get variables from a URL?